### DISCUSSION TITLE
Windows Docker

### DISCUSSION THREAD

**Discussion by schmidiij** [lloesche/valheim-server-docker#28] [Ideas]
Hello :-) this is a really nice work!

Is it possible to create this docker für Windows Docker?

**Comment by InB4DevOps** [comment]
With WSL2 (Windows Subsystem for Linux) built into Windows 10 what for would you want or need it on Windows?

**Reply by schmidiij** [reply]
Now i run the server on a Windows Server 2019.
But with the Solution the user must be sign in.

So i will try / move to the "Windows Docker Solution"

**Reply by CyberT3C** [reply]
I am running a Valheim Dedicated Server under Windows with Docker without any problems at all. Works with Hypver-V as good as it works with WSL2

Why is there any need to sign in? I cannot follow the discussion here. Please explain the context.
🤦 Oh my bad. You want a solution without signing in. - edit
